### PR TITLE
v1.3

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: activities-examples/src/workflows.test.ts [hunk 1]
```diff
@@ -12,19 +12,15 @@ beforeAll(async () => {
     logger: new DefaultLogger('WARN', (entry: LogEntry) => console.log(`[${entry.level}]`, entry.message)),
   });
 
-  testEnv = await TestWorkflowEnvironment.create({
-    testServer: {
-      stdio: 'inherit',
-    },
```
Comment (lorensr) [inline activities-examples/src/workflows.test.ts:18]: @bergundy I forget—is this now default behavior, or is there another equiv option?

Comment (bergundy) [reply activities-examples/src/workflows.test.ts:18]: There’s no output from the test server so this doesn’t matter in practice, for temporalite we allow configuring the log level

**Review by bergundy — APPROVED** [review]
I’m debating whether to change the samples to use the new Client instead

**Comment by lorensr** [comment]
I don't think it's important. It's simple to use `WorkflowClient` when that's the only client you use. In cases where you're using multiple clients, I'd use `Client`.

**Pull request merged**